Userland/Libraries/LibWeb/Layout/TableFormattingContext.cpp

```diff
@@ -152,107 +152,177 @@ void TableFormattingContext::calculate_row_column_grid(Box const& box)
     m_columns.resize(x_width);
 }
 
-void TableFormattingContext::compute_table_measures()
+void TableFormattingContext::compute_table_measures(TableDimension dimension, AvailableSpace const& available_space)
```

Comment:
can we separate part of `compute_table_measures` that does measurements for cells and leave it to just take care of rows/columns depending on passed dimension?

Reply:
Done, I think.

Reply:
it should be possible to calculate all metrics for table cells in one pass so we don't need to call it twice for rows and columns separately.

Reply:
Done.